clients/admin-ui/cypress/support/stubs.ts

```diff
@@ -560,6 +560,10 @@ export const stubActionCenter = () => {
   cy.intercept("PATCH", "/api/v1/plus/discovery-monitor/*/results", {
     response: 200,
   }).as("patchAssets");
+
+  cy.intercept("GET", "/api/v1/plus/system-assets/*", {
+    fixture: "systems/system_assets",
+  }).as("getSystemAssets");
```

Comment:
feels like this is the wrong place for this. I would consider creating a new stub for this tab instead?